src/components/license/VideoLicenseList.tsx

```diff
@@ -89,7 +89,7 @@ const VideoLicenseInfo = ({ video }: VideoLicenseInfoProps) => {
             </AddResourceToFolderModal>
           )}
         </MediaListLicense>
-        {video.cover && <Image alt={video.title} src={video.cover} />}
+        {video.cover && <StyledMediaImage alt={video.title} src={video.cover} />}
```

Comment:
Her kan vi sikkert definere en litt mindre fallbackWidth for å spare noen kb med data

Reply:
Gjelder forsåvidt de andre stedene denne komponenten brukes også.

Reply:
det er ikke brukt så mange steder, er 300 en grei verdi?